tests/AArch64/Tests.S

```diff
@@ -254,9 +254,12 @@ SYMBOL(__aarch64_test_table_begin):
     0x7ff0000000000000 /* inf */, 0x7ff8000000000000, /* nan */              \
     0x7ff8000000000000 /* nan */, 0x7ff0000000000000 /* inf */
 
+#include "tests/AArch64/DATAXFER/LD2_ASISDLSE_R2.S"
+#include "tests/AArch64/DATAXFER/LD2_ASISDLSEP_R2_R.S"
+#include "tests/AArch64/DATAXFER/LD2_ASISDLSEP_I2_I.S"
 
+#if 0
```

Comment:
Doesn't this `#if 0` cause all of the below tests to be skipped?

Reply:
Yes. I should re-enable it!